cli/src/cmd/forge/script/mod.rs

```diff
@@ -445,6 +447,13 @@ impl VerifyBundle {
         let num_of_optimizations =
             if config.optimizer { Some(config.optimizer_runs) } else { None };
 
+        /*let retry =
+            if*/ // TODO: handle default value logic
+        let retry = RetryArgs {
+            retries: 1,
+            delay: None
+        };
```

Comment:
This is to be changed in the future, I just put it to test if it was working, and it seems that it don't because I don't see `retries` in the args nor can I use it.